tests/testthat/test-sql-query.R

```diff
@@ -4,3 +4,14 @@ test_that("add_suffixes works if no suffix requested", {
   expect_equal(add_suffixes(c("x", "x"), "y", ""), c("x", "x"))
   expect_equal(add_suffixes(c("x", "y"), "y", ""), c("x", "y"))
 })
+
+test_that("select_query() print method output is as expected", {
+  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
+  expect_equal(
```

Comment:
Can you please use `expect_known_output()` here instead?

Reply:
`expect_known_output()` does not seem to like the test

``` 
library(testthat)
library(dbplyr)

test_that("select_query() print method output is as expected", {
  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
  expect_known_output(
    paste0(
      capture.output(dbplyr:::print.select_query(mf)),
      collapse = " "
    ),
    "<SQL SELECT> From:     df Select:   *"
  )
})
#> Error: Test failed: 'select_query() print method output is as expected'
#> * cannot open the connection
#> 1: expect_known_output(paste0(capture.output(dbplyr:::print.select_query(mf)), collapse = " "), 
#>        "<SQL SELECT> From:     df Select:   *") at <text>:6
#> 2: write_lines(act$out, file)
#> 3: base::writeLines(enc2utf8(text), path, useBytes = TRUE)
#> 4: file(con, "w")
```

<sup>Created on 2019-01-02 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>